content/en/Platform Deep Dive/Scans/reduced-scope.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "Partial Scans: Reduced Scope"
+linkTitle: "Partial Scans: Reduced Scope"
+weight: 50
+description: Define a specific set of URLs within your target to scan
+---
+
+Rather than scanning the entire target, specify a subset of URLs within your target to scan by defining a Reduced Scope in your target settings. This is particularly useful when you want to focus on specific areas of your application, such as new features or high-risk areas.
+
+### Benefits
+
+- **Faster Feedback:** Get faster feedback on the security of your code changes. This is especially helpful in CI/CD pipelines, where you need to deliver code changes quickly and reliably.
+- **Reduced Scan Time:** Scanning a smaller subset of your target can significantly reduce the overall scan time.
+- **Focused Scans:** Focus your scans on specific areas of your target that are most important or that have undergone recent changes.
+- **Reduced Noise:** Reduce the number of vulnerabilities reported, making it easier to focus on the most critical issues.
+
+### Using a Reduced Scope Partial Scan
+
+1. Go to your target's settings.
+2. In the Scanner tab, locate the **Reduced Scope** section.
+3. Add the URLs you want to include in your partial scan.
+   - Use wildcards (`*`) to include all sub-pages of a specific URL. For example, `https://www.example.com/blog/*` would include all pages within the blog section.
+4. Add and enable as many URLs as you need to define the intended reduced scope.
+   {{% image src="/deepdive/scans/partial-scans-configured.png" alt="Configured Reduced Scope" %}}
+6. Save your changes.
+7. Select **Scan Now** for the target to start a Partial Scan.
+{{% image src="/deepdive/scans/partial-scans-modal.png" alt="Reduced Scope Scan Now" %}}
```

Comment:
The two modals look huge on the page. I suggest using a smaller screen width when taking screenshots in general, and especially for modals.